attest/activation.go

```diff
@@ -160,28 +215,25 @@ func (p *ActivationParameters) checkTPM20AKParameters() error {
 	}
 
 	// Check the signature over the attestation data verifies correctly.
-	pk := rsa.PublicKey{E: int(pub.RSAParameters.Exponent()), N: pub.RSAParameters.Modulus()}
-	signHash, err := pub.RSAParameters.Sign.Hash.Hash()
+	// TODO: Support ECC certifying keys
+	pk := rsa.PublicKey{E: int(vrfy.RSAParameters.Exponent()), N: vrfy.RSAParameters.Modulus()}
+	signHash, err := vrfy.RSAParameters.Sign.Hash.Hash()
 	if err != nil {
 		return err
 	}
 	hsh := signHash.New()
-	hsh.Write(p.AK.CreateAttestation)
-	verifyHash, err := pub.RSAParameters.Sign.Hash.Hash()
```

Comment:
Why did this go away? It's not clear to me what's being verified or why.

Reply:
Before, it was hardcoded to verify self-signed attestation. Now it is the same logic by the signature is verified against CertyfingKey.
`verifyHash` was redundant, wasn't it?